mrp_bom_widget_section_and_note_one2many/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2022 - Today: GRAP (http://www.grap.coop)
+# @author: BADEP
+# @author: Quentin Dupont (quentin.dupont@grap.coop)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "MRP Widget Section and Note in BoM",
+    "summary": "Add section and note in Bills of Materials",
+    "version": "12.0.1.1.0",
+    "category": "GRAP - Incubator",
```

Comment:
I suppose this one is internal GRAP category and Manufacturing/Manufacturing should go there instead

Reply:
:pray:  :heavy_check_mark: